kgx/cli/cli_utils.py

```diff
@@ -280,8 +280,6 @@ def transform(
     output_format: Optional[str] = None,
     output_compression: Optional[str] = None,
     stream: bool = False,
-    node_filters: Optional[Tuple] = None,
-    edge_filters: Optional[Tuple] = None,
     transform_config: str = None,
```

Comment:
@RichardBruskiewich - were these intended to be removed with this 'doc change' PR?

Reply:
Good morning @sierra-moxon, thanks for pointing this change out.

So sorry... I initially pegged these two arguments as unused but as you know, the other day, based on the SonarCloud warning of another sort, I realized that these arguments simply needed to be properly propagated down the stack to where they are referenced. I actually fixed this in SonarCloud patch commit https://github.com/biolink/kgx/pull/298/commits/f698197f7e9bc622282cbc6de61603b8d44ab08b of the streaming PR #298. I should have split the patch off as a separate PR or remembered this PR and merged it here.

So yes, this code change should be ignored. The above commit now takes precedence. Good catch!